lib/puppet/provider/group/windows_adsi.rb

```diff
@@ -9,6 +9,13 @@
 
   has_features :manages_members
 
+  attr_accessor :deleted
```

Comment:
I don't think we need an accessor as this is private state to the provider. Same with user provider below.